src/libstore/s3.hh

```diff
@@ -29,6 +30,16 @@ struct S3Helper
 
     FileTransferResult getObject(
```

Comment:
We left the old `getObject` function around because it's used in `filetransfer.cc`.